app/models/users.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ] Since all these models don't have their reference to the `Users` table as `user` but as `author`, we need to tell Rails that the foreign key of this relationship is under the alias of `author`, like this:

```suggestion
  has_many :posts, foreign_key: 'author_id'
  has_many :comments, foreign_key: 'author_id'
  has_many :likes, foreign_key: 'author_id'
```

Reply:
Thank you for your kind feedback, working on it @anagudelogu.